graphql_compiler/compiler/expressions.py

```diff
@@ -704,8 +704,19 @@ def to_gremlin(self):
         raise NotImplementedError()
 
     def to_cypher(self):
-        """Not supported yet."""
-        raise NotImplementedError()
+        """Return a unicode object with the Cypher representation of this expression."""
+        self.validate()
+
+        _, field_name = self.fold_scope_location.get_location_name()
+        mark_name = u'collected_' + self.fold_scope_location.get_full_path_location_name()
```

Comment:
I think we decided to implement this differently and that code has been merged into `master`, so I think this PR needs an update.